todo_app/user.py

```diff
@@ -39,7 +50,15 @@ def decorator(*args, **kwargs):
                     condition = user.role == Roles.admin if level == 'admin' else user.role != Roles.reader
                     if condition:
                         return fn(*args, **kwargs)
-                
+
+                app.logger.info({
```

Comment:
I notice that this is a WARNING in your screenshot, is there any particular reason it was changed to info?

Reply:
I did initially set it as warning but I wasn't really sure it should be warning? It could signal that someone is trying things they shouldn't so maybe should be a warning?